src/runtime/graph/debug/graph_runtime_debug.cc

```diff
@@ -38,6 +38,39 @@ class GraphRuntimeDebug : public GraphRuntime {
     return time;
   }
 
+  /*!
+   * \brief Run each operation in the graph and print out the runtime per op.
+   * \param warmup Number of warmup runs.
+   * \param iter Number of iteration runs.
+   */
+  void RunIndividual(int warmup, int iter) {
+    for (int i = 0; i < warmup; i++) {
+      GraphRuntime::Run();
+    }
+    std::vector<double> time_per_op(op_execs().size(), 0);
+    for (int k = 0; k < iter; k++) {
+      for (size_t i = 0; i < op_execs().size(); ++i) {
```

Comment:
Should we only measure operators and exclude input and weight nodes? It looks that there is no need to looping on `op_execs()[i] == nullptr`.

Reply:
You're right